tests/qnn/test_cost.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+import pennylane as qml
+
+
+np.random.seed(0)
+
+
+def rx_ansatz(phis, **kwargs):
+    for w, phi in enumerate(phis):
+        qml.RX(phi, wires=w)
+
+
+def layer_ansatz(weights, x=None, **kwargs):
+    qml.templates.AngleEmbedding(x, wires=[0, 1, 2])
+    qml.templates.StronglyEntanglingLayers(weights, wires=[0, 1, 2])
+
+
+class TestMSECost:
```

Comment:
It might be good to add a test that loops over the different types of interface, which could be done with
```python
@pytest.mark.parametrize("interface", qml.qnodes.decorator.ALLOWED_INTERFACES)
```